src/ggml-cpu/vec.h

```diff
@@ -0,0 +1,806 @@
+// Vectorized functions for fundamental operations
+
+#pragma once
+
+#include "ggml-impl.h"
+#include "simd-mappings.h"
+#include "ggml.h"
+
+#include <assert.h>
+
+#define UNUSED GGML_UNUSED
```

Comment:
This macro can be defined per source file (instead of here in the header), as a shortcut for `GGML_UNUSED`. But I think it would be better to just keep using `GGML_UNUSED` to avoid clashes, since `UNUSED` seems like something that could be defined in some other header.

Anyway, this is very minor and no changes needed for this PR.

Reply:
Done, thanks! Used `GGML_UNUSED` in `vec.cpp` and `ops.cpp`, moved this define back into `ggml-cpu.c`.